### PR TITLE
Plat 9575 update bug snag casing in unity plugin UI

### PR DESCRIPTION
**Pull request by clr182**
## Goal

Update from old branding (Bugsnag) to new branding (BugSnag)

## Changeset

Ui Elements were updated to keep inline with BugSnags new branding

## Testing

Tested changes by importing package into the new unity project

### CODE REVIEW

**Comment by clr182** [comment]
Sub task PR: https://github.com/bugsnag/dashboard-js/pull/7093

File: features/fixtures/maze_runner/Assets/Scripts/Scenarios/Csharp/Config/IsStarted.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
```
Comment (richardelms) [inline features/fixtures/maze_runner/Assets/Scripts/Scenarios/Csharp/Config/IsStarted.cs:1]: Why is the IsStarted stuff in this PR?

File: upm-tools/AssemblyDefinitions/Bugsnag.asmdef [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ﻿{
```
Comment (richardelms) [inline upm-tools/AssemblyDefinitions/Bugsnag.asmdef:1]: if you didnt already, please follow the steps in the CONTRIBUTING.md to build a local UPM package and check that this change doesnt break the package

File: upm-tools/AssemblyDefinitions/BugsnagEditor.asmdef [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (richardelms) [inline upm-tools/AssemblyDefinitions/BugsnagEditor.asmdef:1]: if you didnt already, please follow the steps in the CONTRIBUTING.md to build a local UPM package and check that this change doesnt break the package

**Review by richardelms — CHANGES_REQUESTED** [review]

**Pull request closed without merging**